airflow/providers/amazon/aws/log/s3_task_handler.py

```diff
@@ -120,8 +120,9 @@ def _read(self, ti, try_number, metadata=None):
         try:
             log_exists = self.s3_log_exists(remote_loc)
         except Exception as error:  # pylint: disable=broad-except
-            self.log.exception(error)
-            log = f'*** Failed to verify remote log exists {remote_loc}.\n{str(error)}\n'
+            log = f'*** Failed to verify remote log exists {remote_loc}.'
+            self.log.exception(log)
+            log += f'\n{error}\n'
```

Comment:
Slight tweak to this one please (just because the `log` object and the log message appear in different contexts.)

```suggestion
            self.log.exception("Failed to verify remote log exists %s.", remote_loc)
            log = f'*** Failed to verify remote log exists {remote_loc}.'
            log += f'\n{error}\n'
```